hooks/post_gen_project.py

```diff
@@ -0,0 +1,17 @@
+from shutil import copyfile
+from os import listdir
+
+base_path_libs = '../libs'
+
+alldir = listdir(base_path_libs)
```

Comment:
This fails if the `libs` directory doesn't exist, it's best to put it in a try/except block and do nothing if the directory isn't there.